### ISSUE TITLE
[NBug] 入力文字列の形式が正しくありません。

### ISSUE THREAD

**Issue by sxp200422** [gitextensions/gitextensions#8999]
:warning: The sections below must be filled in and this text must be removed or the issue will be closed.

## Error Details
```
System.FormatException: 入力文字列の形式が正しくありません。
   場所 System.Number.StringToNumber(String str, NumberStyles options, NumberBuffer& number, NumberFormatInfo info, Boolean parseDecimal)
   場所 System.Number.ParseInt32(String s, NumberStyles style, NumberFormatInfo info)
   場所 GitUI.Editor.Diff.DiffLineNumAnalyzer.Analyze(String diffContent)
   場所 GitUI.Editor.Diff.DiffViewerLineNumberControl.DisplayLineNumFor(String diff)
   場所 GitUI.Editor.FileViewerInternal.SetText(String text, Action openWithDifftool, Boolean isDiff)
   場所 GitUI.Editor.FileViewer.<>c__DisplayClass103_0.<ViewPatchAsync>b__0()
   場所 GitUI.Editor.FileViewer.ShowOrDeferAsync(Int64 contentLength, Func`1 showFunc)
   場所 GitUI.Editor.FileViewer.<ViewPatchAsync>d__103.MoveNext()
--- 直前に例外がスローされた場所からのスタック トレースの終わり ---
   場所 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   場所 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   場所 GitUI.CommandsDialogs.RevisionDiffControl.<ShowSelectedFileDiffAsync>d__35.MoveNext()
--- 直前に例外がスローされた場所からのスタック トレースの終わり ---
   場所 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   場所 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   場所 GitUI.CommandsDialogs.RevisionDiffControl.<<DiffText_ExtraDiffArgumentsChanged>b__39_0>d.MoveNext()
--- 直前に例外がスローされた場所からのスタック トレースの終わり ---
   場所 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   場所 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   場所 GitUI.ThreadHelper.<>c__DisplayClass13_0.<<FileAndForget>b__0>d.MoveNext()
```

## Additional information
encode error

## Environment
- Git Extensions 3.4.3.9999
- Build d4b0f48bbf3e39a71f5d7ff5231be5678d4aa0f1
- Git 2.27.0.windows.1
- Microsoft Windows NT 10.0.18363.0
- .NET Framework 4.8.4180.0
- DPI 96dpi (no scaling)

**Comment by RussKie** [comment]
I believe this was fixed, and caused by use of UTF7.
Please try the latest dev version.

**Comment by gerhardol** [comment]
If this is caused in the same as #8634, it was fixed in #8661
Previous occurrences with explanation has been related to use of UTF-7 (that is removed in 3.5)

Duplicate of #7146

**Comment by RussKie** [comment]
Presume fixed. If not, please provide a standalone repro on the latest versions, and we'll reopen.
